ledgers-bank-account-access-management/ledgers-bank-account-access-repository/src/main/java/de/adorsys/ledgers/baam/db/domain/HolderAccess.java

```diff
@@ -0,0 +1,55 @@
+package de.adorsys.ledgers.baam.db.domain;
+
+import jakarta.persistence.*;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "holder_access")
+@Data
+@NoArgsConstructor(force = true)
+public class HolderAccess extends BankAccountAccess {
+
+
+//    @Column(name = "holder_id")
+    @Column(name = "holder_id")
+    private String  holderId; // ID of the account holder
+
+    @Column(name = "user_id", nullable = false)
+    private String userId;
+
+
+
+    // Constructor for creating HolderAccess with full access
+    public HolderAccess(String holderId) {
+        super(); // Call to the superclass constructor
+        this.holderId = holderId;
+        this.userId = userId;
```

Comment:
i don't see any need for adding the ```super()``` here, i guess you have your reasons..

also  these ```this.userId = userId``` is cool but not proper..

before initializing, you should always pass it as a **parameter inside the constructor*** just like you did for the ```holderId```